httpcore5/src/main/java/org/apache/hc/core5/util/Deadline.java

```diff
@@ -58,12 +58,12 @@
     /**
      * The maximum (longest-lived) deadline.
      */
-    public static Deadline MAX_VALUE = new Deadline(INTERNAL_MAX_VALUE);
+    public static final Deadline MAX_VALUE = new Deadline(INTERNAL_MAX_VALUE);
 
     /**
      * The minimum (shortest-lived) deadline.
      */
-    public static Deadline MIN_VALUE = new Deadline(INTERNAL_MIN_VALUE);
+    public static final Deadline MIN_VALUE = new Deadline(INTERNAL_MIN_VALUE);
```

Comment:
@arturobernalg I am afraid this will could as an API / binary compatibility breakage. Could you please run `mvn clirr:check` and see that it passes with your changes?

Reply:
True. Reverted the changes